lms/static/sass/features/_course-experience.scss

```diff
@@ -171,3 +171,21 @@
     }
   }
 }
+
+// Course Updates Page
+div.course-updates {
+  div.all-updates {
+      article.updates-article {
+        margin: 10px 0px;
```

Comment:
We use a variable ``$baseline`` for pixel sizes which default to 20px. Also, units aren't needed on sizes that are set to 0. So this should be:

    margin: ($baseline/2) 0;